app/components/ProfileModal/index.js

```diff
@@ -86,27 +88,40 @@ const ProfileModal = ({
     },
   ];
 
+  console.log('avatar', avatar);
+
   return (
     <Modal visible={isProfileModalVisible} transparent>
       <View style={styles.container}>
         {userInfo.email && (
           <View style={styles.subContainer}>
             <View style={styles.headerContainer}>
               <TouchableOpacity
+                style={styles.closeButtonContainer}
                 onPress={onPressCloseProfileModal}
                 accessible={true}
                 accessibilityLabel="Profile Modal"
                 testID="profile_modal">
                 <Ionicons name={'md-close'} size={30} color={Colors.TEXT_COLOR} />
               </TouchableOpacity>
-              <SvgXml xml={logo} />
-              <View />
+              <View style={styles.logoContainer}>
+                <SvgXml xml={logo} />
+              </View>
             </View>
             <View style={styles.profileSection1}>
-              <Image
-                style={{ width: 50, height: 50, marginHorizontal: 10, borderRadius: 50 }}
-                source={{ uri: avatar }}
-              />
+              {avatar ? (
```

Comment:
`avatar` probably is never undefined because of the default image is still used at https://github.com/Plant-for-the-Planet-org/treemapper/blob/85b08bb4893e32788a10790cd240436f42caccfc/app/components/ProfileModal/index.js#L36-L38

Reply:
ohh thanks. Fixed it